staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -1084,6 +1084,10 @@ type PodNetwork struct {
 	// CIDR for vm network.
 	// Default 10.0.2.0/24 if not specified.
 	VMNetworkCIDR string `json:"vmNetworkCIDR,omitempty"`
+
+	// IPv6 CIDR for the vm network.
+	// Defaults to fd10:0:2::/120 if not specified.
+	VMIpv6NetworkCIDR string `json:"vmIpv6NetworkCIDR,omitempty"`
```

Comment:
It looks like this is not used anywhere. Is there maybe a commit missing?

Reply:
Originally I wanted to add a test, see it fails and then link it to the usage and see it passes.
But it seems I need to do some work on the network tests in general and introduce tests for these API/s afterwards.

I added the usage now, not to confuse anyone.